coralproject/talk
# Feat/corl 2158 invalid embedded media urls

## What does this PR do?

This PR addresses CORL-2158 by rendering appropriate responses and HTTP codes when handling oembed requests for external media.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

Paste an invalid url for external media (Twitter, Giphy, Youtube, or other) and verify that the preview renders friendly message rather than JSON.
